pokedex_async_redux/lib/utilities/constants.dart

```diff
@@ -0,0 +1,3 @@
+export 'custom_constant.dart';
```

Comment:
the thing with this wild card export is that, when you import this constants.dart file then all of its components will be taken , which seems not efficient especially if I only need is the string_constant 🙂